content/blog/tools-of-our-own.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Tools of Our Own"
+date = 2020-05-14
+description = "What is a digital environment? What does it mean to shape your own digital environment?"
+[extra]
+author = "Matilde Park"
+ship = "~haddef-sigwen"
+image =""
++++
+
+<br>
+
+<img class="ba" src="https://media.urbit.org/site/posts/essays/tools-of-our-own-1.png">
+
+## Tools of Our Own
```

Comment:
Cut

Reply:
Are you specifically noting to removed the `## Tools of Our Own`?

Reply:
Yeah, it's not meant as a section title, it's the title title.